### DISCUSSION TITLE
5.0.11 でページリストが20件までしか表示できない

### DISCUSSION THREAD

**Discussion by TatsuyaIse** [growilabs/growi#6465] [Help / Questions] [answered]
GROWI のバージョン 5.0.11 以降で確認できましたが、ページリストの機能を使って表示するとページの一覧が20件までしか表示されません。
これは意図した仕様でしょうか？
また `$lsx()` をページに記載する以外に、配下のページ一覧を画面から閲覧する方法はありますか？
→ 元の質問は Slack の #help チャンネルにて: https://wsgrowi.slack.com/archives/C4V5ZG4UC/p1660880403918179

**Comment by yuki-takei** [comment, marked as answer]
バグです。v5.1.3 で修正されました。
https://github.com/weseek/growi/pull/6468
